DOCS-10430 missing feature (#202)
mulesoft/docs-api-community-manager: modules/ROOT/pages/troubleshooting.adoc

== Duplicated External Objects

Validating and synchronizing the external data source generates duplicate external objects that will not be used. Delete these new objects.

== Missing Feature Error

> If installation of Anypoint API Community Manager shows a `Missing feaure` error, ensure digital experiences (previously known as communities) are enabled in your API Community Manager Salesforce organization.

Reviewer: ```suggestion
If installation of Anypoint API Community Manager results in a `Missing feaure` error, ensure that digital experiences (previously known as _communities_) are enabled in your API Community Manager Salesforce organization.
```

Reviewer: ...or c/b active voice: If you see the error <blah> after you install....

> Example:

Reviewer: GLOBALLY avoid using only "For example:" or "Example:" to lead in to an example.
Either end the sentence preceding with a colon, or say something like "The following example shows...:"

```suggestion
Here is an example of how this error looks in the interface:
```

----
1. Lightning Community Themes(communityThemeDefinitions/acm_template.communityThemeDefinition) Missing feature
Installing this package requires the following feature and its associated permissions: Lightning Community Themes
2. Lightning Community Templates(communityTemplateDefinitions/NTO.communityTemplateDefinition) Missing feature
Installing this package requires the following feature and its associated permissions: Lightning Community Templates
3. Navigation Link Sets(navigationMenus/acm_template_Nav0.navigationMenu) Missing feature
Installing this package requires the following feature and its associated permissions: Navigation Link Sets
----

> To fix this error, follow the installation prerequisite steps in the section xref:installation-prerequisites.adoc#set-up-a-domain-with-enabled-communities[set up a domain with enabled communities].

Reviewer: Since we are using the text as a title, title-style caps are OK:
```suggestion
To fix this error, follow the installation prerequisite steps in the section xref:installation-prerequisites.adoc#set-up-a-domain-with-enabled-communities[Set Up a Domain with Enabled Communities].
```
